department-of-veterans-affairs/caseflow
# Reassign Case page Task(s) not displaying

Resolves [APPEALS-24904](https://jira.devops.va.gov/browse/APPEALS-24904), [APPEALS-27506](https://jira.devops.va.gov/browse/APPEALS-27506)

# Description
Task(s) to be cancelled do not display under the "The following task(s) will be cancelled" on the Reassign Case Page 

## Acceptance Criteria
- [ ] Task(s) to be cancelled will display under the "The following task(s) will be cancelled" on the Reassign Case Page 

## Testing Plan
- Login to: Moon Walkers Demo Environment
- Change user to [BVARDUNKLE (VACO)](https://mw.caseflowdemo.com/queue/appeals/082250264#BVARDUNKLE%20(VACO))
- Select task action dropdown
- On Reassign Case Page error is shown given task(s) are not displaying under the "The following task(s) will be cancelled" 

![imagen11](https://github.com/department-of-veterans-affairs/caseflow/assets/110848569/9421cddc-4f3f-4350-ba67-7167ca85d3cd)
